authnz/src/main/java/io/stargate/auth/entity/ResourceKind.java

```diff
@@ -23,5 +23,6 @@
   TRIGGER,
   AGGREGATE,
   VIEW,
-  INDEX
+  INDEX,
+  UDT
```

Comment:
We have `TYPE` on line 22... How is `UDT` different from the auth perspective?

Reply:
I mean that we do not currently distinguish UDTs in CQL auth... If we want to make it a distinct auth resource kinds, we'd probably have to fix CQL auth too (`StargateQueryHandler`).

Reply:
+1 for using `TYPE`